PATCHES/gcc.cfg

```diff
@@ -0,0 +1,17 @@
+# Change gcc bug report
+spec_change {
+  search_and_replace {
+    any: true
+    find: "http://bugzilla.redhat.com/bugzilla"
+    replace: "https://bugs.openela.org/"
```

Comment:
You are replacing URL with no / in the end with URL with /, if there is an original URL with /, you will end up with double /.
I did not check the code though, and it might not be an issue if all original URLs are without /.

Reply:
Not an issue, http://bugzilla.redhat.com/bugzilla is provided as an URL to configure 
	--with-bugurl=http://bugzilla.redhat.com/bugzilla \

Reply:
so I guess we can consider it resolved?